src/views/optin.php

```diff
@@ -2,118 +2,47 @@
-		font-size: 18px;
-		font-weight: 500;
-		line-height: 24px;
-		letter-spacing: -0.02em;
-		color: #000;
-	}
-
-	.stellarwp-telemetry {
-		font-family: 'SF Pro Text', 'Helvetica Neue', 'Helvetica', 'Arial', sans-serif;
-		font-size: 14px;
-		font-weight: 400;
-		line-height: 18px;
-		color: #4E4E4E;
-	}
-
-	.stellarwp-telemetry .btn-primary,
-	.stellarwp-telemetry .btn-text {
-		padding: 6px 12px;
-		background: #0047FF;
-		border-radius: 4px;
-		color: #fff;
-		text-decoration: none;
-		border: 0;
-		outline: none;
-		transition: all 0.1s ease-in-out;
-		cursor: pointer;
-	}
-
-	.stellarwp-telemetry .btn-primary:hover {
-		background: #0032b7;
-	}
-
-	.stellarwp-telemetry .btn-text {
-		background: #fff;
-		color: #000;
-	}
-
-	.stellarwp-telemetry .links a:hover,
-	.stellarwp-telemetry .btn-text:hover {
-		text-decoration: underline;
-	}
-
-	.stellarwp-telemetry .links {
-		display: flex;
-		gap: 16px;
-		font-size: 12px;
-		line-height: 16px;
-	}
-
-	.stellarwp-telemetry .links a {
-		color: #0047FF;
-		text-decoration: none;
-	}
-</style>
-
-<div id="modal-wrapper" class="stellarwp-telemetry wrapper">
-	<section class="stellarwp-telemetry modal">
+<div class="stellarwp-telemetry stellarwp-telemetry-modal stellarwp-telemetry-modal--active" data-js="optin-modal">
+	<section class="stellarwp-telemetry-modal__inner">
 		<header>
-			<img src="<?php echo $args['plugin_logo']; ?>" width="<?php echo $args['plugin_logo_width']; ?>"
-			     height="<?php echo $args['plugin_logo_height']; ?>" alt="<?php echo $args['plugin_logo_alt']; ?>"/>
-			<h1 class="stellarwp-telemetry-header">We hope you love <?php echo $args['plugin_name']; ?>.</h1>
+			<img src="<?php echo esc_url( $args['plugin_logo'] ); ?>" width="<?php echo esc_attr( $args['plugin_logo_width'] ); ?>" height="<?php echo esc_attr( $args['plugin_logo_height'] ); ?>" alt="<?php echo esc_attr( $args['plugin_logo_alt'] ); ?>"/>
+			<h1 class="stellarwp-telemetry__title">
+				<?php echo esc_attr( $args['heading'] ); ?>
+			</h1>
 		</header>
 		<main>
-			<p>Hi, <?php echo $args['user_name']; ?>! This is an invitation to help our StellarWP community.
-				If you opt-in, some data about your usage of <?php echo $args['plugin_name']; ?> and future
-				StellarWP Products will be shared with our teams (so they can work their butts off to improve).
-				We will also share some helpful info on WordPress, and our products from time to time.
-				And if you skip this, that’s okay! Our products still work just fine.</p>
-			<ul class="links">
-				<li><a href="<?php echo $args['permissions_url']; ?>" target="_blank">What permissions are being
-						granted?</a></li>
-				<li><a href="<?php echo $args['tos_url']; ?>" target="_blank">Terms of Service</a></li>
-				<li><a href="<?php echo $args['privacy_url']; ?>" target="_blank">Privacy Policy</a></li>
+			<p>
+				<?php echo esc_attr( $args['intro'] ); ?>
+			</p>
+			<ul class="stellarwp-telemetry-links">
```

Comment:
Are these three links a requirement? Or should add them to the filter as an array similar to the uninstall reasons?